runtime-modules/blog/src/benchmarking.rs

```diff
@@ -0,0 +1,392 @@
+    let min_handle_length = 1;
+
+    let mut handle = vec![];
+
+    for i in 0..4 {
+        handle.push(get_byte(id, i));
+    }
+
+    while handle.len() < (min_handle_length as usize) {
+        handle.push(0u8);
+    }
+
+    handle
+}
+
+fn generate_post<T: Trait<I>, I: Instance>() -> PostId {
+    assert_eq!(Blog::<T, I>::post_count(), 0);
+
+    Blog::<T, I>::create_post(RawOrigin::Root.into(), vec![0u8], vec![0u8]).unwrap();
+
+    let post_id = 0;
+
+    assert_eq!(Blog::<T, I>::post_count(), 1);
+
+    assert!(Blog::<T, I>::post_by_id(post_id) == Post::<T, I>::new(&vec![0u8], &vec![0u8]));
+
+    post_id
+}
+
+fn generate_reply<T: Trait<I>, I: Instance>(
+    creator_id: T::AccountId,
+    participant_id: ParticipantId<T>,
+    post_id: PostId,
+) -> T::ReplyId {
+    let creator_origin = RawOrigin::Signed(creator_id);
+    Blog::<T, I>::create_reply(
+        creator_origin.clone().into(),
+        participant_id,
+        post_id,
+        None,
+        vec![0u8],
+    )
+    .unwrap();
+
+    assert!(
+        Blog::<T, I>::reply_by_id(post_id, T::ReplyId::zero())
+            == Reply::<T, I>::new(vec![0u8], participant_id, ParentId::Post(post_id))
+    );
+
+    T::ReplyId::zero()
+}
+
+benchmarks_instance! {
+    where_clause { where T: balances::Trait, T: membership::Trait }
+
+    _ {}
+
+    create_post {
+        let t in 0 .. MAX_BYTES;
+        let b in 0 .. MAX_BYTES;
+        assert_eq!(Blog::<T, I>::post_count(), 0);
+        let title = vec![0u8; t.try_into().unwrap()];
+        let body = vec![0u8; b.try_into().unwrap()];
+
+    }:_(RawOrigin::Root, title.clone(), body.clone())
+    verify {
+        assert_eq!(Blog::<T, I>::post_count(), 1);
+
+        assert!(
+            Blog::<T, I>::post_by_id(0) ==
+            Post::<T, I>::new(&title, &body)
+        );
+
+        assert_last_event::<T, I>(RawEvent::PostCreated(
+                0,
+                title,
+                body
+            ).into());
+    }
+
+    lock_post {
+        let post_id = generate_post::<T, I>();
+    }: _(RawOrigin::Root, post_id)
+    verify {
+        assert!(Blog::<T, I>::post_by_id(post_id).is_locked());
+        assert_last_event::<T, I>(RawEvent::PostLocked(post_id).into());
+    }
+
+    unlock_post {
+        let post_id = generate_post::<T, I>();
+        Blog::<T, I>::lock_post(RawOrigin::Root.into(), post_id).unwrap();
+        assert!(Blog::<T, I>::post_by_id(post_id).is_locked());
+    }: _(RawOrigin::Root, post_id)
+    verify {
+        assert!(!Blog::<T, I>::post_by_id(post_id).is_locked());
+        assert_last_event::<T, I>(RawEvent::PostUnlocked(post_id).into());
+    }
+
+    edit_post {
+        let t in 0 .. MAX_BYTES;
+        let b in 0 .. MAX_BYTES;
+
+        let post_id = generate_post::<T, I>();
+        let title = Some(vec![1u8; t.try_into().unwrap()]);
+        let body = Some(vec![1u8; b.try_into().unwrap()]);
+    }: _(
+        RawOrigin::Root,
```

Comment:
Do we need such formatting in this and other blog benchmarks? It looks different from the `cargo fmt` style.